src/gui/widgets/homewidget.cpp

```diff
@@ -40,9 +40,17 @@ void HomeWidget::on_pushButton_clicked()
 {
     using namespace experimental;
     DB()->connect();
-    //DataPosition dp = {"pilots", 7};
-    //DEB(DB()->getEntryDataNew(dp));
-    long customFunc = 0;
+
+    auto entry = DB()->getPilotEntry(7);
+    DEB(entry.position);
+    DEB(entry.getData());
+
+    PilotEntry copiedEntry;
+    copiedEntry = entry;
+    DEB(copiedEntry.position);
+    DEB(copiedEntry.getData());
+
+    /*long customFunc = 0;
```

Comment:
Did you perhaps forget to cleanup debugging?

Reply:
indeed xD